bsdlib/include/nrf_socket.h

```diff
@@ -1003,6 +1018,27 @@ typedef uint16_t nrf_gnss_nmea_mask_t;
  *          - @c NRF_GNSS_PSM_DUTY_CYCLING_PERFORMANCE for low power mode with better performance.
  *          - @c NRF_GNSS_PSM_DUTY_CYCLING_POWER for low power mode with lower power consumption.
  *
+ * The GNSS receiver may track signals (i.e. run) either continuously or in duty
```

Comment:
This information is now here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/bsdlib/doc/gnss_extension.html

We discussed this in the previous PR , please do not put conceptual documentation into API documentation.

Reply:
yes, there was an earlier backport issue, will do fixes and take your suggestions to that. Thanks!